tests/sharingTest.js

```diff
@@ -0,0 +1,703 @@
+////////////////////////////////
+///////    MAIN TESTS    ///////
+////////////////////////////////
+
+/* globals ownCloud, __karma__ */
+
+// CURRENT TIME
+var timeRightNow = new Date().getTime();
```

Comment:
I suggest moving all those vars into the first `describe()` block. There's a risk that these will conflict with the values from other files if loaded in sequence.

Reply:
in this case I need to duplicate all these vars into two describe blocks - right?

Reply:
oh, I didn't noticed that. Then wrap the whole file in another `describe()` block then that describes the whole file...